crypto/fipsmodule/aes/asm/aes-gcm-avx2-x86_64.pl

```diff
@@ -57,7 +57,12 @@
   or ( $xlate = "${dir}../../../perlasm/x86_64-xlate.pl" and -f $xlate )
   or die "can't locate x86_64-xlate.pl";
 
-open OUT, "| \"$^X\" \"$xlate\" $flavour \"$output\"";
+my $xlate_binutils;
+( $xlate_binutils = "${dir}xlate-old-binutils.pl" and -f $xlate_binutils )
+  or ( $xlate_binutils = "${dir}../../../perlasm/xlate-old-binutils.pl" and -f $xlate_binutils )
+  or die "can't locate xlate-old-binutils.pl";
```

Comment:
I like your thinking here but this is likely to break some build systems I am aware of but am not allowed to explain to you because it adds a new file dependency. My suggestion is that we change back to the inline approach from the previous version and then develop a more general approach in the avx512 version where we'll have more time. I will do a release this morning if we can get this going. I will comment in the avx512 issue about the more general approach.

Thanks!

Reply:
k